interoperability_report.py

```diff
@@ -426,15 +435,18 @@ def run_test(
                                 'subscribers_finished':subscribers_finished,
                                 'publisher_finished':publishers_finished[num_publishers]
             }))
-            num_publishers+=1
+            num_publishers += 1
```

Comment:
```suggestion
            num_publisher += 1
```

or even `publisher_number`

Reply:
But this variable means the total number of publishers

Reply:
done